components/cards/cards.css

```diff
@@ -0,0 +1,113 @@
+.card-container{
+    /* background-color: var(--primary-color); */
+    margin: 1rem;
+    padding: 1rem;
+    border-radius: 0.8rem;
+    box-shadow: 0px 0px 10px var(--secondary-color);
+    position: relative;
+}
+.vertical{
+    max-height: 48rem;
+    max-width: 25rem;
+}
+.horizontal{
+    display: flex;
+    width: 45rem;
+    max-height: 25rem;
+}
+.card-img{
+    border-radius: 0.5rem;
+    width: 10rem;
+    height: 10rem;
+}
+.vertical-img{
+    width: 100%;
+    height: 25rem;
+    border-bottom: 0.3rem solid black;
+}
+.horizontal-img{
+    width: 25rem;
+    height: 100%;
+    border-right: 0.3rem solid black;
+}
+img{
```

Comment:
we shouldn't set any property directly to any html tag. It will apply on every element(same type) rendered on DOM. Sometimes we don't need to set width/height 100% to every image(like in this case).

Reply:
ok, I will look into it. Thank you